polyply/src/topology.py

```diff
@@ -252,6 +266,7 @@ def add_positions_from_file(self, path):
                    meta_mol.molecule.nodes[node]["position"] = position
                    meta_mol.molecule.nodes[node]["build"] = False
                    total += 1
+                   last_atom = total
```

Comment:
What's going on here?

Reply:
this is needed in case all positions are actually defined. Previously it would raise an error because last_atom wasn't defined